pyproject.toml

```diff
@@ -34,6 +34,7 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
+# These match the suffices on the various requirements-*.txt files
```

Comment:
Should this be `suffixes` instead of `suffices`?

Reply:
You learn something new every day!